public_html/js/pinform.js

```diff
@@ -270,7 +271,8 @@ PIN.Form.getParams = function(args) {
 
 // main public method
 PIN.Form.render = function(args) {
-
+    //comment:  point - 2
+    console.log("called render");
```

Comment:
prefix all console calls with `PIN.Form.DEBUG && ` so that it is easy to toggle debugging mode on/off with a single var.